Update prerequisites for HCP Vault Radar Copy Secrets to Vault (#1414)
hashicorp/web-unified-docs: content/hcp-docs/content/docs/vault-radar/remediate-secrets/copy-secrets.mdx

- The [HCP Vault Radar agent v0.27.0 or greater is running](/hcp/docs/vault-radar/agent/deploy) and configured
- A HashiCorp Vault Enterprise or HCP Vault Dedicated cluster [configured for the Vault Radar Agent](/hcp/docs/vault-radar/agent/correlate-vault)

> - A Data Source onboarded to HCP Vault Radar

Reviewer: ```suggestion
- A data source onboarded to HCP Vault Radar
```

- The [HCP Vault Radar agent v0.27.0 or greater is running](/hcp/docs/vault-radar/agent/deploy) and configured
- A HashiCorp Vault Enterprise or HCP Vault Dedicated cluster [configured for the Vault Radar Agent](/hcp/docs/vault-radar/agent/correlate-vault)
- A data source onboarded to HCP Vault Radar
      
## Configure HCP Vault Radar Data Sources

There are some additional steps necessary to enable secret copying for HCP configured data sources.
1. Edit the data source configuration in the HCP UI.
![Edit Data Source Details](/img/docs/vault-radar/remediation/copy-secrets/edit-data-source-details.png)

1. Check the box at the bottom of the data source details that says "Enable secret copying via Vault Radar Agent".
![Enable Secret Copying via Vault Radar Agent](/img/docs/vault-radar/remediation/copy-secrets/enable-secret-copying-via-vault-radar-agent.png)

1. Provide the URI to an environment variable that contains a credential used to access the data source.

>    - Note: This variable will need to be set on ALL of the agent workers!

Reviewer: ```suggestion
   <Note>
   
   You need to set this variable on all agent workers.
   
   </Note>
```
